components/prism-bash.js

```diff
@@ -113,6 +113,12 @@
 			alias: 'variable',
 			lookbehind: true
 		},
+		// Highlight parameter names as variables
+		'parameter': {
+			pattern: /(^|\s)-{1,2}(\w+:[+-]?)?\w+(?:\.\w+)*(?=[=\s]|$)/,
```

Comment:
Tests fail because of this unnecessary capturing group.

```suggestion
			pattern: /(^|\s)-{1,2}(?:\w+:[+-]?)?\w+(?:\.\w+)*(?=[=\s]|$)/,
```